x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/index.ts

```diff
@@ -4,16 +4,12 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export { PipelineProcessorsContextProvider, Props } from './context';
```

Comment:
I removed the exports that are no longer being used. I think we can easily add back if there ends up being a need.